english/marshmallow-attack/make-smores/make-smores.md

```diff
@@ -0,0 +1,7 @@
+Mmmmm! S'mores are tasty!  You can't wait to take a bite of the gooey chocolately crispy goodness!
+
+But wait -- do you have chocolate and graham crackers?
+
+What? No Chocolate? [Look in your pockets]  (look-for-chocolate/look-for-chocolate.md)
+
+OMG!  No chocolate AND no graham crackers [Start to cry] (/start-to-cry/start-to-cry.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.